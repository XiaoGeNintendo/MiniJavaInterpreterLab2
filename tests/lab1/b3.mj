int main(){
    int a=0;
    a++;
    a++;
    ++a;
    a++;
    --a;
    {
        int a=114514;
        int b=344555;
        {
            string a="114514";
            println(a);
        }
        println(a);
        println(b);
    }
    println(a);
    return 0;
}