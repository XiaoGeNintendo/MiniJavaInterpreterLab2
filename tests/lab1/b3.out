114514
114514
344555
3
Process exits with the code 0.